features/features.go

```diff
@@ -538,6 +538,14 @@ var (
 						enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
+	FeatureGateNewOLMOwnSingleNamespace = newFeatureGate("NewOLMOwnSingleNamespace").
+						reportProblemsToJiraComponent("olm").
+						contactPerson("thetechnick").
```

Comment:
Generally we use the kerberos ID rather than the GH handle for these

Reply:
Nico is on PTO this week. I've addressed your comment but had to clone the branch to #2264. 
Closing this PR in favor of that.